ImageSegmentation/README.md

```diff
@@ -46,6 +46,36 @@ Select an iOS simulator or device on Xcode to run the app. The example image and
 
 Note that the `resized` method in `UIImage+Helper.swift` is used to speed up the model inference, but a smaller size may cause the result to be less accurate.
 
+## Using the Lite/Mobile Interpreter Model
+
+All the other iOS demo apps have been converted to use the new Mobile Interpreter model, except this Image Segmentation demo app, which is used to illustrate how to convert a demo using a full JIT model to one using the mobile interpreter model, by following 3 simple steps.
+
+### 1. Prepare the Lite model
+
+If you don't have the PyTorch environment set up to run the script `deeplabv3.py` to generate the mobile interpreter model file, you can download it to the `ios-demo-app/ImageSegmentation` folder using the link [here](https://pytorch-mobile-demo-apps.s3.us-east-2.amazonaws.com/deeplabv3_scripted.ptl). Otherwise, or if you prefer to run the script to generate the model yourself, just run `python deeplabv3.py`.
+
+Note that to save a model in the mobile interpreter format, simply call `_save_for_lite_interpreter`, as shown at the end of the `deeplabv3.py`:
+```
+optimized_model.save("ImageSegmentation/deeplabv3_scripted.pt")
+optimized_model._save_for_lite_interpreter("ImageSegmentation/deeplabv3_scripted.ptl")
+```
+
+### 2. Modify the Podfile
+
+If you already went through the previous section and have the demo using the full JIT model up and running, close Xcode, go to the `ios-demo-app/ImageSegmentation` directory and run `pod deintegrate` first.
+
+In `Podfile`, change `pod 'LibTorch', '~>1.9.0'` to `pod 'LibTorch-Lite', '~>1.9.0'`
+
+Then run `pod install` and `open ImageSegmentation.xcworkspace`. Don't forget to drag and drop the `deeplabv3_scripted.ptl` file from step 1 to the project.
+
+### 3. Change the iOS code
+
+In `InferenceModule.mm`, first change `#import <LibTorch/LibTorch.h>` to `#import <Libtorch-Lite/Libtorch-Lite.h>`, then change `@protected torch::jit::script::Module _impl;` to `@protected torch::jit::mobile::Module _impl;` and `_impl = torch::jit::load(filePath.UTF8String);` to `_impl = torch::jit::_load_for_mobile(filePath.UTF8String);`.
```

Comment:
why @protected?

Reply:
i got it from the HelloWorld example last year i think and never really thought about it - even tutorials like https://pytorch.org/tutorials/recipes/mobile_interpreter.html refer to the protected - should we change it or leave it for consistency?